tools/fonts/inspector/glyphs.html

```diff
@@ -96,54 +159,65 @@ <h1 id="title">Glyph Inspector</h1>
         ctx.setTransform(2, 0, 0, 2, 0, 0); // reset the scale to (2, 2)
 
         var x = 0;
-        var y = 50;
-        for (let glyphName in glyphs) {
+        var y = 250;
+
+        const arr = Object.keys(SMuFLGlyphInfo).sort(function(a,b) { 
+          return parseInt(SMuFLGlyphInfo[a].codepoint.substring(2), 16) - parseInt(SMuFLGlyphInfo[b].codepoint.substring(2), 16); 
+        } );
+        arr.forEach((glyphName, index) => {
           // Show the glyph name.
+          const code = parseInt(SMuFLGlyphInfo[glyphName].codepoint.substring(2), 16);
           ctx.fillStyle = '#222';
+          ctx.strokeStyle = '#222';
           ctx.font = '8pt Arial';
-          ctx.fillText(glyphName, x, y + 30);
-
-          const originX = x + 100;
+          if (glyphName.length > 25) {
+            ctx.fillText(glyphName.slice(0,22)+'...', x, y + 50);
```

Comment:
Comment here might be nice. It looks like you're breaking it up into two lines?

Reply:
done